doc/theory/glossary.tex

```diff
@@ -339,30 +494,54 @@
 	\begin{equation*}	
 	||\lambda_i||_p = \left( \int_{\mathbb R} \lambda_i^p(x)\, dx \right)^{1/p}	
 	\end{equation*}	
-	is finite. In this case we refer to \eqref{equation:persistence_landscape_norm} as the
-	%\hyperref[vectorization_kernel_and_amplitude]{amplitude}
-	\textit{landscape} $p$-\textit{amplitude} of $D$.
+	is finite. In this case we refer to \eqref{equation:persistence_landscape_norm} as the $p$-\textit{landscape norm} of $D$ and, for $p = 2$, define the value of the \textit{landscape kernel} on two persistence diagrams $D$ and $E$ as
+	\begin{equation*}
+	\langle \lambda, \mu \rangle = \left(\sum_{i \in \mathbb N} \int_{\mathbb R} |\lambda_i(x) - \mu_i(x)|^2\, dx\right)^{1/2}
+	\end{equation*}
+	where $\lambda$ and $\mu$ are their associated persistence landscapes.
 
 	\paragraph{\\ References:} \cite{bubenik2015statistical}
 	
 	\subsection*{Weighted silhouette} \label{weighted_silhouette}
 	
 	Let $D = \{(b_i, d_i)\}_{i \in I}$ be a
 	%\hyperref[persistence_diagram]{persistence diagram}
-	persistence diagram and $w = \{w_i\}_{i \in I}$ a set of positive real numbers. The \textit{silhouette of $D$ weighted by $w$} is the function $\phi : \mathbb R \to \mathbb R$ defined by	
+	persistence diagram and $w = \{w_i\}_{i \in I}$ a set of positive real numbers. The \textit{silhouette of} $D$ \textit{weighted by} $w$ is the function $\phi : \mathbb R \to \mathbb R$ defined by
 	\begin{equation*}	
 	\phi(t) = \frac{\sum_{i \in I}w_i \Lambda_i(t)}{\sum_{i \in I}w_i},	
 	\end{equation*}	
 	where	
 	\begin{equation*}
 	\Lambda_i(t) = \left[ \min \{t-b_i, d_i-t\}\right]_+	
 	\end{equation*}	
-	and $c_+ := \max(c,0)$. When $w_i = \vert d_i - b_i \vert^p$ for $0 < p \leq \infty$ we refer to $\phi$ as the \textit{$p$-power-weighted silhouette} of $D$. The silhouette construction defines a
-	%\hyperref[vectorization_kernel_and_amplitude]{vectorization}
-	vectorization of the set of persistence diagrams with target the vector space of continuous real-valued functions on $\mathbb R$.	
+	and $c_+ := \max(c,0)$. When $w_i = \vert d_i - b_i \vert^p$ for $0 < p \leq \infty$ we refer to $\phi$ as the $p$-\textit{power-weighted silhouette} of $D$. The silhouette construction defines a
+	%\hyperref[vectorization_amplitude_and_kernel]{vectorization}
+	vectorization of the set of persistence diagrams with target the vector space of continuous real-valued functions on $\mathbb R$.
 	
 	\paragraph{\\ References:} \cite{chazal2014stochastic}
 	
+	\subsection*{Heat vectorizations} \label{heat_vectorization}
+	
+	Considering the points in a persistence diagram as the support of Dirac deltas one can construct, for any $t > 0$,
+	%\hyperref[vectorization_amplitude_and_kernel]{vectorization}
+	two vectorizations of the set of persistence diagrams to the set of continuous real-valued function on the first quadrant $\mathbb{R}^2_{>0}$. The \textit{symmetry heat vectorization} is constructed for every persistence diagram $D$ by solving the heat equation
```

Comment:
I thought we had settled for "antisymmetric" over "symmetric" in https://github.com/giotto-ai/giotto-tda/pull/348#discussion_r391668768. Maybe that discussion got superseded?